job-service-container/src/test/java/com/hpe/caf/services/job/api/QueueManager.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2016-2021 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hpe.caf.services.job.api;
+
+import java.io.Closeable;
+import java.io.IOException;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.LinkedBlockingQueue;
+import java.util.concurrent.TimeoutException;
+import org.apache.commons.lang3.StringUtils;
+
+import com.hpe.caf.api.CodecException;
+import com.hpe.caf.api.worker.QueueTaskMessage;
+import com.hpe.caf.util.rabbitmq.DefaultRabbitConsumer;
+import com.hpe.caf.util.rabbitmq.Event;
+import com.hpe.caf.util.rabbitmq.QueueConsumer;
+import com.hpe.caf.util.rabbitmq.RabbitUtil;
+import com.hpe.caf.worker.testing.QueueServices;
+import com.hpe.caf.worker.testing.WorkerServices;
+import com.rabbitmq.client.Channel;
+import com.rabbitmq.client.Connection;
+import com.rabbitmq.client.MessageProperties;
+
+public class QueueManager implements Closeable
```

Comment:
As discussed these are copied from [`worker-testing-util`](https://github.com/WorkerFramework/worker-framework/tree/v5.1.0/worker-testing-util/src/main/java/com/hpe/caf/worker/testing).  It's fine if you want to merge this now but let's rationalize this to get these out again before this Jira is over.